packages/teleport/src/Nodes/AddNode/useAddNode.ts

```diff
@@ -49,14 +49,14 @@ export default function useAddNode(ctx: TeleportContext) {
 
   return {
     createJoinToken,
+    isEnterprise,
```

Comment:
is this flag still needed? (just looking at the backport PR it was removed)

Reply:
Good catch, but yeah in this fork the `Manually` component still requires the `isEnterprise` flag whereas in `master` it doesn't so it wasn't a direct backport.